nengo_loihi/splitter.py

```diff
@@ -92,7 +94,7 @@ def __init__(self, dimensions, neuron_type=None):
         super(ChipReceiveNeurons, self).__init__(dimensions, dimensions)
 
 
-def split(model, inter_rate, inter_n):  # noqa: C901
+def split(model, loihi_model):  # noqa: C901
```

Comment:
The splitter accepting `loihi_model` feels like it will make the splitter harder to test once we start to write tests for it. If we're only using the model to get `max_rate` and `dt`, are you ok with modifying this to pass those in instead? Currently also the `dt` is only being used to raise a build error -- this definitely feels like it should be done in the builder and not in the splitter, in which case `split` would only need to accept `max_rate`, which is much easier to test.

Reply:
Ok, fair. In #95 I end up using `inter_tau` here, too, but I guess we can just pass those two (and there might be a way to default that, too, so we don't have to pass it).